linera-chain/src/manager.rs

```diff
@@ -149,7 +149,7 @@ impl ChainManager {
                 .values()
                 .map(|(_, weight)| *weight)
                 .collect();
-            Some(WeightedAliasIndex::new(weights)?)
+            Some(WeightedAliasIndex::new(weights).map_err(ChainError::OwnerWeightError)?)
```

Comment:
This change is needed to support `wasm32-unknown-unknown` in its full generality.  Since `linera-execution` now needs a feature branch, I guess this also branches anyway so we may as well remove this change :)